core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -95,10 +95,14 @@ public static String getSymbol(String name, String title, String tooltip, String
             String symbol = symbolsForLookup.get(translatedName);
             symbol = symbol.replaceAll("(class=\")[^&]*?(\")", "$1$2");
             symbol = symbol.replaceAll("(tooltip=\")[^&]*?(\")", "");
+            symbol = symbol.replaceAll("(data-html-tooltip=\")[^&]*?(\")", "");
```

Comment:
```suggestion
            symbol = symbol.replaceAll("(data-html-tooltip=\").*?(\")", "");
```

Mostly noting, between https://github.com/jenkinsci/jenkins/pull/6989 and https://github.com/jenkinsci/jenkins/pull/6659 it's probably not needed.